composer.json

```diff
@@ -59,6 +61,11 @@
     }
   },
   "extra": {
+    "patches": {
+        "friends-of-behat/mink-extension": {
+            "setClient error from PR16": "https://github.com/FriendsOfBehat/MinkExtension/files/9480850/16.txt"
```

Comment:
even with the version constraint, a patch is IMHO too unstable, sooner or later that won't apply anymore or will be merged. We could maybe combine it with a specific requirement on that specific version, but waiting for that to be merged and committed is probably the safer option.

Reply:
@Berdir there have been a number of requests to merge the PR and it hasn't happened, so I don't fundamentally disagree with the concern, but I'm not sure if the maintainer can merge things at the moment given their location. we did try and make this "more stable" by pointing at a patch file vs. the actual PR. i guess we could explicitly pin to 2.7.1 to prevent accidental breakage / updates?